src/main/java/com/EstimatedDeliveryDate/Entity/CriticalPullTime.java

```diff
@@ -0,0 +1,46 @@
+package com.EstimatedDeliveryDate.Entity;
+
+import java.util.List;
+
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBAttribute;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBHashKey;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBTable;
+
+/*EDD_CPT table item is mapped to CriticalPullTime object*/
+
+@DynamoDBTable(tableName = "EDD_CPT")
+public class CriticalPullTime {
+	private String SHIPPING_SERVICE_ID;
+	private List<CutOffTimeList> PROCESSING_DAY_CUTOFF_LIST;
+	private List<String> DELIVERY_DAYS;
```

Comment:
Java naming conventions should be followed (https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html). You can map the DDB fields to Java objects using some library like Jackson and DDB also provides DTO mappers.

Reply:
The DynamoDBMapper class is mapping Java objects to DDB fields in Repository calss

Reply:
@DynamoDBAttribute provides you to map the fields to any hava fields you have in the class.
Please refer to https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/DynamoDBMapper.Annotations.html#DynamoDBMapper.Annotations.DynamoDBAttribute